example/l2fwd_simple/odp_l2fwd_simple.c

```diff
@@ -204,6 +206,23 @@ int main(int argc, char **argv)
 	global.if1 = create_pktio(argv[optind + 1], pool, &global.if1in,
 								&global.if1out);
 
+	/* Do some operations to increase code coverage in tests */
+	if (odp_pktio_mac_addr(global.if0, &correct_src, sizeof(correct_src))
+	    != sizeof(correct_src))
+		printf("Warning: can't get MAC address\n");
+	else if (memcmp(&correct_src, &global.src, sizeof(correct_src)) != 0)
+		printf("Warning: src MAC invalid\n");
+
+	odp_pktio_promisc_mode_set(global.if0, true);
+	odp_pktio_promisc_mode_set(global.if1, true);
+	(void)odp_pktio_promisc_mode(global.if0);
+	(void)odp_pktio_promisc_mode(global.if1);
+
+	mtu1 = odp_pktio_mtu(global.if0);
+	mtu2 = odp_pktio_mtu(global.if1);
```

Comment:
`odp_pktio_mtu()` is a deprecated API and should not be used in new code.

Reply:
Ok, the only reason I used this call was to increase code coverage. But is it deprecated? I grepped through the repository, finding to evidence that any API having the name mtu would be deprecated...

Reply:
Ah, it is deprecated in api-next. My patch, however, is to master as it has no API changes.

Reply:
At this point, since we're tagging Tiger Moth RC1 this week, all of api-next is being merged into master, so this will be on that base.